scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -68,7 +68,11 @@ check-try-runtime:
         echo "Found label runtimemigration. Running tests"
         export RUST_LOG=remote-ext=debug,runtime=debug
         echo "---------- Running try-runtime for ${NETWORK} ----------"
-        time cargo run --release --features=try-runtime try-runtime --chain=${NETWORK}-dev --execution=Wasm --no-spec-check-panic on-runtime-upgrade live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
+        time cargo build --release --features try-runtime
+        time cargo run --release --features try-runtime try-runtime \
+            --runtime ./target/release/wbuild/runtime.wasm -lruntime=debug \
```

Comment:
`./target` assumes that `$CARGO_TARGET_DIR` is unset. If this project is using rusty-cachier, it means that `$CARGO_TARGET_DIR` is defined because rusty-cachier sets it to a custom location. Maybe you can use something like this:

```bash
--runtime "${CARGO_TARGET_DIR:-$PWD/target}/release/wbuild/runtime.wasm"
```

Reply:
Nevermind the rusty-cachier part. I thought it was being used because of https://github.com/paritytech/polkadot/blob/6e3f2c5b4b6e6927915de2f784e1d831717760fa/scripts/ci/gitlab/pipeline/test.yml#L84, but apparently not.